content/WebAuthn/WebAuthn_Developer_Guide/WebAuthn_Client_Registration.adoc

```diff
@@ -68,19 +68,22 @@ _Note: In the WebAuthn API, all binary parameters (`user.id`, `challenge` etc.)
 
 **user:** contains data about the user account. The **displayName** provides a user-friendly name which may be displayed to the user. The **name** attribute takes a traditional username or the like, to disambiguate accounts with similar **displayNames**. The **id** attribute is the user handle that is generated by the RP and used to uniquely identify a user. Unlike **name**, **id** is not meant to be human-readable and should not contain personally identifying information such as an e-mail address.
 
-**2.** The JavaScript client calls `navigator.credentials.create()`. The browser validates the `rp.id` against the origin, hashes the `clientData`, and calls `authenticatorMakeCredential` method.
+**Step 2:** The JavaScript client calls `navigator.credentials.create()`. The browser validates the `rp.id` against the origin, hashes the `clientData`, and calls `authenticatorMakeCredential` method.
 
-**3.** Before proceeding, the authenticator will ask for some form of user consent. After verifying consent, the authenticator will create a new asymmetric key pair and safely store the private key. The public key becomes part of the attestation, which the authenticator signs over with the attestation private key. The manufacturer’s attestation certificate chain may also be returned so that the relying party can validate the device back to a root of trust.
+_Note: When you retrieve the public key credentials, even if you use the same object to generate the registration, `MakeCredential` will make new credentials every time unless you use the exclude list to pass in the list of previously created credentials. See `excludeCredentials` above._
```

Comment:
I think this might be clearer as something like:

```suggestion
_Note: Even with the same parameter object, `navigator.credentials.create()` will create a new credential every time it is called. Use the exclude list to prevent registering the same authenticator twice. See `excludeCredentials` above._
```